<ui version="4.0" >
 <class>SynchronisationSettingsDlg</class>
 <widget class="QWidget" name="SynchronisationSettingsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QCheckBox" name="kcfg_AskForSync" >
     <property name="text" >
      <string>Nachfragen vor Synchronisation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="title" >
      <string>Verfügbare Sprachmodelle</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Hier können Sie zu einer älteren Modellversion zurückkehren.</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lwModels" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="KPushButton" name="pbLoadList" >
          <property name="text" >
           <string>Liste verfügbarer Modelle laden</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="pbSelectModel" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Modell auswählen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>tableView</zorder>
     <zorder>pbSelectModel</zorder>
     <zorder>horizontalSpacer</zorder>
     <zorder>lwModels</zorder>
     <zorder>label</zorder>
     <zorder>pbSelectModel</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
